android/src/main/java/com/robinpowered/react/ScreenBrightness/ScreenBrightnessModule.java

```diff
@@ -61,13 +61,16 @@ public String getName() {
         return MODULE_NAME;
     }
 
-    @Override
-    public void onActivityResult(int requestCode, int resultCode, Intent intent) {
+    public void onActivityResult(Activity activity, int requestCode, int resultCode, Intent data) {
         if (requestCode == writeSettingsRequestCode) {
             onPermissionResult();
         }
     }
 
+    public void onNewIntent(Intent intent) {
+        // void
```

Comment:
`// no-op` is more of a typical comment if we're intentionally not doing anything